CRM/Campaign/Form/Petition/Signature.php

```diff
@@ -123,6 +123,18 @@ class CRM_Campaign_Form_Petition_Signature extends CRM_Core_Form {
    */
   public $submitOnce = TRUE;
 
+  protected array $forceEmailConfirmed;
```

Comment:
Should define as an array I think since in `__construct` we kinda just add to it - that might be fine but if feels like it might not be guaranteed to work on all php versions

```suggestion
  protected array $forceEmailConfirmed = [];
```